lib/ansible/modules/cloud/vmware/vmware_host_ntp_info.py

```diff
@@ -124,9 +125,11 @@ def main():
         ],
         supports_check_mode=True,
     )
+    if module._name == 'vmware_host_ntp_facts':
+        module.deprecate("The 'vmware_host_ntp_facts' module has been renamed to 'vmware_host_ntp_info'", version='2.13')
 
     vmware_host_ntp_config = VmwareNtpFactManager(module)
```

Comment:
This is internal to the module and not visible to the outside, so it can also be changed later. I think it's more important to first fix everything visible to the user.